specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2022-08-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,8364 @@
+            "$ref": "./examples/Glossary_ListTermsByGlossaryName.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/glossaryName"
+          },
+          {
+            "$ref": "#/parameters/limit"
+          },
+          {
+            "$ref": "#/parameters/offset"
+          },
+          {
+            "$ref": "#/parameters/includeTermHierarchy"
+          },
+          {
+            "$ref": "#/parameters/extInfo"
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. An array of glossary terms for the given glossary or an empty list.",
+            "schema": {
+              "type": "array",
+              "description": "An array of glossary terms for the given glossary or an empty list.",
+              "items": {
+                "$ref": "#/definitions/AtlasGlossaryTerm"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/glossary/terms/bulkDeletion/{operationGuid}": {
+      "get": {
+        "tags": [
+          "Glossary"
+        ],
+        "description": "Get the status of bulk delete operation",
+        "operationId": "Glossary_GetBulkDeleteOperationStatus",
+        "x-ms-examples": {
+          "Glossary_GetBulkDeleteOperationStatus": {
+            "$ref": "./examples/Glossary_GetBulkDeleteOperationStatusByOperationguid.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/operationGuid"
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Return the status of bulk delete operation.",
+            "schema": {
+              "$ref": "#/definitions/BulkDeleteOperation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/glossary/terms:delete": {
+      "post": {
+        "tags": [
+          "Glossary"
+        ],
+        "description": "Delete Glossary Terms",
+        "operationId": "Glossary_BulkDeleteTermsByGuids",
+        "x-ms-examples": {
+          "Glossary_BulkDeleteTermsByGuids": {
+            "$ref": "./examples/Glossary_GetBulkDeleteOperationStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/forceDeleteEntityAssignment"
+          },
+          {
+            "$ref": "#/parameters/forceDeleteChild"
+          },
```

Comment:
nit, one thing to think about boolean is whether one day you'd wish to have an enum here for other than true/false.